### PR TITLE
[plugin.video.longnow] v0.1.2

### PR DESCRIPTION
**Pull request by desperadoduck**
### Description
minor changes to provide matrix compatibility.
### Checklist:
- [x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.longnow/addon.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_gb">Long Now Seminar Viewer</summary>
+    <description lang="en_gb">This program is an unofficial viewer for seminars of the Long Now Foundation in Kodi. You need to have an account at https://longnow.org to make use of that plugin.</description>
+    <disclaimer lang="en_gb">This is not an official plugin of the Long Now Foundation, but my private project, so please don't bother them if anything breaks.</disclaimer>
+    <language>en</language>
+    <platform>all</platform>
+    <license>GPL-2.0-or-later</license>
+    <forum></forum>
```
Comment (enen92) [inline plugin.video.longnow/addon.xml]: drop if empty

File: plugin.video.longnow/addon.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <website>http://www.galois.de/projects/kodi_longnow</website>
+    <email></email>
```
Comment (enen92) [inline plugin.video.longnow/addon.xml]: drop if empty

File: plugin.video.longnow/changelog.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+v0.1.2 (2020-12-28)
```
Comment (enen92) [inline plugin.video.longnow/changelog.txt:1]: note: the changelog is no longer parsed by kodi. You might want to define the `<news>` element in addon xml addon metadata

File: plugin.video.longnow/resources/language/English/strings.po [hunk 4]
```diff
@@ -0,0 +1,60 @@
+msgid ""
```
Comment (enen92) [inline plugin.video.longnow/resources/language/English/strings.po]: language folder layout is wrong: resources/language/resource.language.en_gb/strings.po

**Review by enen92 — CHANGES_REQUESTED** [review]
see comments

File: plugin.video.longnow/addon.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+<addon id="plugin.video.longnow" name="Long Now Seminars" version="0.1.2" provider-name="desperadoduck">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.beautifulsoup4" version="4.8.2" />
+    <import addon="script.module.requests" version="2.22.0" />
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
```
Comment (enen92) [inline plugin.video.longnow/addon.xml:11]: lang="en_GB", check travis failures

Comment (bull-florian-schmitt) [reply plugin.video.longnow/addon.xml:11]: fixed

File: plugin.video.longnow/changelog.txt (hunk 3 above)
Comment (bull-florian-schmitt) [reply plugin.video.longnow/changelog.txt:1]: thanks, I added a news section as you suggested.

File: plugin.video.longnow/addon.xml (hunk 1 above)
Comment (bull-florian-schmitt) [reply plugin.video.longnow/addon.xml]: done

File: plugin.video.longnow/addon.xml (hunk 2 above)
Comment (bull-florian-schmitt) [reply plugin.video.longnow/addon.xml]: done

**Comment by desperadoduck** [comment]
I think I fixed everything.

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
Thanks for the submission

**Pull request merged**